openedx/core/djangoapps/user_api/views.py

```diff
@@ -474,6 +532,36 @@ def _add_password_field(self, form_desc, required=True):
             required=required
         )
 
+    def _add_password_copy_field(self, form_desc, required=True):
+        """Add a password copy field to a form description.
+
+        Arguments:
+            form_desc: A form description
+
+        Keyword Arguments:
+            required (bool): Whether this field is required; defaults to True
+
+        """
+        # Translators: This label appears above a field on the registration form
+        # meant to hold the user's retyped password.
+        password_copy_label = _(u"Retype password")
```

Comment:
I don't like this word `Retype` here. I think many other online registration form uses `Confirm password`. I think we should follow this convention.